src/popup/fxa.js

```diff
@@ -1,11 +1,12 @@
 "use strict";
 
-const ENTRYPOINT = "fxa_discoverability";
+const ENTRYPOINT = "fxa_discoverability_v2";
```

Comment:
@irrationalagent Any objections to changing this to for phase 2? If it doesn't get changed, amplitude would not be able to differentiate the phases, or maybe there is a better way to do this?

Reply:
this LGTM. if we want to collapse all versions together in amplitude you can do something like `entrypoint contains fxa_discoverability`. but its important that they be separable.